acct-user/apt-cacher-ng/metadata.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<!-- maintainer-needed -->
```

Comment:
Well I noticed you put yourself as maintainer on the group package, but not user?

Reply:
Whoops, that is by mistake, sorry! https://github.com/gentoo/gentoo/pull/19414 opened to fix it.